src/OpenSpaceNet.cpp

```diff
@@ -276,6 +281,7 @@ void OpenSpaceNet::initFeatureSet()
     VectorOpenMode openMode = args_.append ? APPEND : OVERWRITE;
 
     featureSet_ = make_unique<FeatureSet>(args_.outputPath, args_.outputFormat, args_.layerName, definitions, openMode);
+    featureSet_->setProjection(sr_.wkt());
```

Comment:
That's from the vector-read PR on Deepcore-Dev. I need to make a PR for changes necessary in OSN for both that and exclusion areas.